projects/packages/forms/src/contact-form/class-contact-form-field.php

```diff
@@ -679,11 +679,13 @@ public function render_radio_field( $id, $label, $value, $class, $required, $req
 	 * @return string HTML
 	 */
 	public function render_checkbox_field( $id, $label, $value, $class, $required, $required_field_text ) {
-		$field  = "<input id='" . esc_attr( $id ) . "' type='checkbox' name='" . esc_attr( $id ) . "' value='" . esc_attr__( 'Yes', 'jetpack-forms' ) . "' " . $class . checked( (bool) $value, true, false ) . ' ' . ( $required ? "required aria-required='true'" : '' ) . "/> \n";
+		$field  = "<div class='contact-form__checkbox-wrap'>";
```

Comment:
Group label and input in a wrapper so that the error message is not added in the same flexbox.